src/components/Link.js

```diff
@@ -13,8 +13,8 @@ import { isPreviewMode } from '../utils/is-preview-mode';
 const Link = React.forwardRef(
     ({ children, to, activeClassName, partiallyActive, ...other }, ref) => {
         if (!to) to = '';
-        // Assume that external links begin with http:// or https://
-        const external = /^http(s)?:\/\//.test(to);
+        // Assume that external links begin with http:// or https:// or have mailto
+        const external = /^http(s)?:\/\/|mailto:[\w-.]+@(\w+.)/.test(to);
```

Comment:
```suggestion
        const external = /^http(s)?:\/\/|^mailto:[\w-.]+@(\w+.)/.test(to);
```

We can also add the `^` here to indicate this should occur at the start.